document/PolicyComplianceProfile.mdk

```diff
@@ -1 +1,65 @@
-# Policy Compliance Profile { #policy-compliance-profile }
\ No newline at end of file
+# Policy Compliance Profile { #policy-compliance-profile }
```

Comment:
Yes, as discussed, I will leave this profile around, but unlink it from the spec (and also eliminate the place where I mention it in the list of existing profiles).